src/app/scripts/cac/control/cac-control-directions.js

```diff
@@ -98,9 +142,6 @@ CAC.Control.Directions = (function (_, $, moment, Control, Geocoder, Routing, Ty
      * Throttled to cut down on requests.
      */
     var planTrip = _.throttle(function() {  // jshint ignore:line
-        if (!tabControl.isTabShowing(tabControl.TABS.DIRECTIONS)) {
-            return;
-        }
```

Comment:
This is mostly redundant, but not entirely.  `setOptions` is being called by a listener in the Home controller and is in turn calling `planTrip` without checking whether the tab is active.  Probably putting the check in `setOptions` makes sense.